.github/workflows/integration-oci.yml

```diff
@@ -105,6 +106,13 @@ jobs:
           path: apps/call_summary_bot
           ref: ${{ matrix.call-summary-bot-versions }}
 
+      - name: Checkout circles app
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
```

Comment:
Any reason to use a different checkout version here then for the other checkouts?

Reply:
Copy pasted but others were updated since